spring-kafka/src/main/java/org/springframework/kafka/retrytopic/DestinationTopic.java

```diff
@@ -68,8 +68,8 @@ public boolean isNoOpsTopic() {
 		return Type.NO_OPS.equals(this.properties.type);
 	}
 
-	public boolean isSingleTopicRetry() {
-		return Type.SINGLE_TOPIC_RETRY.equals(this.properties.type);
+	public boolean isReusableRetryTopic() {
```

Comment:
I think this is a breaking change - we can keep both methods and deprecate the old one.